data_rentgen/server/api/v1/router/job.py

```diff
@@ -57,3 +60,12 @@ async def get_jobs_lineage(
     )
 
     return build_lineage_response(lineage)
+
+
+@router.get("/job_types", summary="Get distinct types of Jobs")
```

Comment:
```suggestion
@router.get("/job-types", summary="Get distinct types of Jobs")
```
For consistency with `personal-tokens`